busy_beaver/tasks/github_stats/summary.py

```diff
@@ -17,24 +17,27 @@
 class GitHubUserEvents:
     def __init__(self, user: User, boundary_dt: datetime):
         self.user = user
-        self.events = {  # this is the order of summary output
-            "releases_published": ReleasesPublishedList(),
-            "created_repos": CreatedReposList(),
-            "publicized_repos": PublicizedReposList(),
-            "forked_repos": ForkedReposList(),
-            "pull_requests": PullRequestsList(),
-            "issues_opened": IssuesOpenedList(),
-            "commits": CommitsList(),
-            "starred_repos": StarredReposList(),
-        }
+        self.event_lists = [  # this is the order of summary output
+            ReleasesPublishedList(),
+            CreatedReposList(),
+            PublicizedReposList(),
+            ForkedReposList(),
+            PullRequestsList(),
+            IssuesOpenedList(),
+            CommitsList(),
+            StarredReposList(),
+        ]
 
         timeline = github.user_activity_after(user.github_username, boundary_dt)
-        self._classify_events(timeline)
+        for event in timeline:
+            for event_list in self.event_lists:
+                if event_list.matches_event(event):
```

Comment:
This is nicely done! Hide all the complexity in your classes and make the rest of your life readable.

Will be using this a lot.